helm/values.yaml

```diff
@@ -182,6 +182,8 @@ mocks:
 cluster:
   kind: Kubernetes # Kubernetes | OpenShift
 
+  ingressNamespace:
```

Comment:
Now, add a `--set` command to the Makefile, so that the networkpolicy is applied for the local setup.